src/Diagnostics.ModelsAndUtils/Attributes/Definition.cs

```diff
@@ -56,17 +57,38 @@ public Definition()
 
         /// <summary>
         /// List of Support Topics for which this detector is enabled.
+        /// Mark it as JsonIgnore because the SupportTopic class is deriving
+        /// from Attribute and attributes are not serialized by System.Text.Json
         /// </summary>
-        [DataMember]
+        [JsonIgnore]
         public IEnumerable<SupportTopic> SupportTopicList { get; set; }
 
-        [JsonIgnore]
+        /// <summary>
+        /// Property created only for Json Serialization as Attributes
+        /// are not serialized today properly by System.Text.Json
+        /// https://github.com/dotnet/runtime/issues/58947
+        /// </summary>
+        [DataMember]
+        [JsonPropertyName("supportTopicList")]
+        public IEnumerable<SupportTopicJson> SupportTopicListJson
```

Comment:
Small nitpik, but maybe rename this as `SupportTopicListSTJ` or `SupportTopicListSTJCompat` since the return type is technically not json type.

Reply:
done